Lib/bdb.py

```diff
@@ -306,7 +306,12 @@ def dispatch_line(self, frame):
         self.user_line(). Raise BdbQuit if self.quitting is set.
         Return self.trace_dispatch to continue tracing in this scope.
         """
-        if self.stop_here(frame) or self.break_here(frame):
+        # GH-136057
+        # For line events, we don't want to stop at the same line where
+        # we issue the previous next/step command.
```

Comment:
```suggestion
        # the latest next/step command was issued.
```

("we" here is the debugger, so it's not we who issued the command).